### PR TITLE
CHE-9284 add tests for messages API

### PR DESCRIPTION
**Pull request by olexii4**
Add tests for messages API.

Signed-off-by: Oleksii Orel <oorel@redhat.com>

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -27,6 +27,57 @@ export function start() {
             }
         })
     );
+
+    const informationMessageTestCommand = {
+        id: 'server-plugin-information-message-command',
```
Comment (benoitf) [inline src/index.ts]: could you rename server by backend ?

File: src/index.ts [hunk 2]
```diff
@@ -27,6 +27,57 @@ export function start() {
...
+        label: "Test Information Message Server Plugin Item"
+    };
+    disposables.push(theia.commands.registerCommand(informationMessageTestCommand, (...args: any[]) => {
+        theia.window.showInformationMessage('Information message!');
+    }));
+
+    const informationModalMessageTestCommand = {
+        id: 'server-plugin-information-modal-message-command',
```
Comment (benoitf) [inline src/index.ts]: here as well

File: src/index.ts [hunk 3]
```diff
@@ -27,6 +27,57 @@ export function start() {
...
+    };
+    disposables.push(theia.commands.registerCommand(informationModalMessageTestCommand, (...args: any[]) => {
+        theia.window.showInformationMessage('Information modal message!', {modal: true}, {title: 'action1'},
+            {title: 'action2', isCloseAffordance: true}, {title: 'action3'}).then(action => {
+            console.log('>>> resolve', action);
+        });
+    }));
+
+    const warningMessageTestCommand = {
+        id: 'server-plugin-warning-message-command',
```
Comment (benoitf) [inline src/index.ts]: and there

File: src/index.ts [hunk 4]
```diff
@@ -27,6 +27,57 @@ export function start() {
...
+    const errorMessageTestCommand = {
+        id: 'server-plugin-error-message-command',
+        label: "Test Error Message Server Plugin Item"
+    };
+    disposables.push(theia.commands.registerCommand(errorMessageTestCommand, (...args: any[]) => {
+        theia.window.showErrorMessage('Error message!');
+    }));
+
+    const errorModalMessageTestCommand = {
+        id: 'server-plugin-error-modal-message-command',
```
Comment (benoitf) [inline src/index.ts]: here

File: src/index.ts [hunk 5]
```diff
@@ -27,6 +27,57 @@ export function start() {
...
+    const warningModalMessageTestCommand = {
+        id: 'server-plugin-warning-modal-message-command',
+        label: "Test Warning Modal Message Server Plugin Item"
+    };
+    disposables.push(theia.commands.registerCommand(warningModalMessageTestCommand, (...args: any[]) => {
+        theia.window.showWarningMessage('Warning modal message!', {modal: true});
+    }));
+
+    const errorMessageTestCommand = {
+        id: 'server-plugin-error-message-command',
```
Comment (benoitf) [inline src/index.ts]: here

File: src/index.ts [hunk 6]
```diff
@@ -27,6 +27,57 @@ export function start() {
...
+    const warningMessageTestCommand = {
+        id: 'server-plugin-warning-message-command',
+        label: "Test Warning Message Server Plugin Item"
+    };
+    disposables.push(theia.commands.registerCommand(warningMessageTestCommand, (...args: any[]) => {
+        theia.window.showWarningMessage('Warning message!');
+    }));
+
+    const warningModalMessageTestCommand = {
+        id: 'server-plugin-warning-modal-message-command',
```
Comment (benoitf) [inline src/index.ts]: here

File: src/index.ts (hunk 1 above)
Comment (olexii4) [reply src/index.ts]: done

File: src/index.ts (hunk 2 above)
Comment (olexii4) [reply src/index.ts]: done

File: src/index.ts (hunk 3 above)
Comment (olexii4) [reply src/index.ts]: done

File: src/index.ts (hunk 4 above)
Comment (olexii4) [reply src/index.ts]: done

File: src/index.ts (hunk 5 above)
Comment (olexii4) [reply src/index.ts]: done

File: src/index.ts (hunk 6 above)
Comment (olexii4) [reply src/index.ts]: done

File: src/index.ts [hunk 7]
```diff
@@ -27,6 +27,57 @@ export function start() {
             }
         })
     );
+
+    const informationMessageTestCommand = {
+        id: 'backend -plugin-information-message-command',
```
Comment (benoitf) [inline src/index.ts]: it seems there is a extra space between backend and plugin all over the file ?

Comment (olexii4) [reply src/index.ts]: A paste typo. Done.

**Review by benoitf — APPROVED** [review]

**Review by evidolob — APPROVED** [review]

**Pull request merged**